cmake/modules/RootBuildOptions.cmake

```diff
@@ -360,7 +360,7 @@ foreach(opt afdsmgrd afs bonjour castor chirp geocad glite globus gviz hdfs ios
 endforeach()
 
 #---Deprecated options---------------------------------------------------------
-foreach(opt memstat)
+foreach(opt memstat vmc)
```

Comment:
Can you add an entry in the release notes for this deprecation (at the top, IIRC)? Please reference the new canonical repo at https://github.com/vmc-project/vmc

Reply:
Ok, thanks for reminding me. I will add as a new commit, so we don't have to test this again.